ThreeDSecure/src/androidTest/java/com/braintreepayments/api/ThreeDSecureClientTest.java

```diff
@@ -59,7 +59,7 @@ public void onResult(JSONObject tokenizationResponse, Exception exception) {
 
                 CardNonce cardNonce = null;
                 try {
-                    cardNonce = new CardNonce(tokenizationResponse);
+                    cardNonce = CardNonce.fromJSON(tokenizationResponse);
```

Comment:
This is unrelated to hungarian notation but fixing a test compiler error missed in in the previous PR